lib/classification_lifecycle.rb

```diff
@@ -14,6 +14,12 @@ def queue(action)
       raise ClassificationNotPersisted.new(message)
     end
     ClassificationWorker.perform_async(classification.id, action)
+    if classification.complete?
+      classification.subject_ids.each do |sid|
+        ClassificationCountWorker.perform_async(sid, classification.workflow_id)
+        RetirementWorker.perform_async(sid, classification.workflow_id)
```

Comment:
Yeah this should suffice as a first attempt and what we need for stargazing. 

Also are you going to add the cellect call if it's retiring as well?